app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people.update({self.name: self})
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for human in people:
+        Person(human["name"], human["age"])
+
+    for human in people:
+        person = Person.people[human["name"]]
+        if "wife" in human and human["wife"]:
```

Comment:
Instead of explicitly checking for the presence of the 'wife' key, use the `dict.get()` method to simplify the code and avoid potential `KeyError` exceptions. This approach is recommended in the checklist.